packages/ui/react-components/src/infostripe/Infostripe.tsx

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import './infostripe.less';
 
-interface InfostripeProps {
-    text: string;
+interface BaseInfostripeProps {
     iconRenderer: () => React.ReactNode;
 }
 
-const Infostripe = ({ text, iconRenderer }: InfostripeProps) => (
+interface ElementInfostripeProps extends BaseInfostripeProps {
+    element: JSX.Element;
+    text?: string;
```

Comment:
Kan muligens sette `never` som type her siden `text` ikke vil bli rendret dersom element er definert

Reply:
Forsøkte det, men får følgende feil. Men er enig i at det blir riktig løsning. Vært borti det?

"Type '{ element: Element; iconRenderer: () => Element; }' is not assignable to type 'IntrinsicAttributes & InfostripeProps'.
  Property 'text' is missing in type '{ element: Element; iconRenderer: () => Element; }' but required in type 'TextInfostripeProps'.ts(2322)
Infostripe.d.ts(11, 5): 'text' is declared here."

Reply:
Det fungerer fint her. Testet å lage en fil i samme mappe som Infostripe og som brukte den med de ulike kombinasjonene